docs/fluvio/concepts/batching.mdx

```diff
@@ -6,10 +6,87 @@ title: "Batching"
 Fluvio producers try to send records in batches to reduce the number of messages sent and improve throughput. Each producer has some configurations that can be set to improve performance for a specific use case. For instance, they can be used to reduce disk usage, reduce latency, or improve throughput.
 As of today, batching behavior in Fluvio Producers can be modified with the following configurations:
 
-- `batch_size`: Indicates the maximum amount of bytes that can be accumulated in a batch.
-- `linger`: Time to wait before sending messages to the server. Defaults to 100 ms.
+- `max_request_size`: Indicates the maximum amount of bytes that the producer can send in a single request. If the record is larger than the max request size, the producer will fail to send the record. Only the uncompressed size of the record is considered. Defaults to 1048576 bytes.
+- `batch_size`: Indicates the maximum amount of bytes that can be accumulated in a batch. If the record is larger than the batch size, the producer will send the record in a single new batch. Only the uncompressed size of the record is considered. Defaults to 16384 bytes.
 - `compression`: Compression algorithm used by the producer to compress each batch before sending it to the SPU. Supported compression algorithms are none, gzip, snappy and lz4.
+- `linger`:  Time to wait before sending batches to the server that have not reached maximum batch size. Defaults to 100 ms.
 
-In general, each one of these configurations has a benefit and a potential drawback. For instance, with the compression algorithm, it is a trade-off between disk usage in the server and CPU usage in the producer and the consumer for compression and decompression. Typically, the compression ratio is improved when the payload is large, therefore a larger `batch_size` could be used to improve the compression ratio. A `linger` equals `0` means that each record is sent as soon as possible. A `linger` time larger than zero introduces latency but improves throughput.
 
-The ideal parameters for the `batch_size`, `linger` and `compression` depend on your application needs.
+# Trade-offs and Considerations
+
+Each configuration offers a balance of benefits and trade-offs:
+
+- `max_request_size`: Allows the producer to send larger records, but may increase the risk of network congestion and latency. 
+- `batch_size`: Larger value can reduce the number of requests sent to the server, but may increase latency. Smaller value can reduce latency, but may increase the number of requests.
```

Comment:
but may increase latency => but `will` increase latency

Remove this: <s>Smaller value can reduce latency, but may increase the number of requests.</s>